eng/pipelines/compliance.yml

```diff
@@ -0,0 +1,80 @@
+resources:
+  pipelines:
+    - pipeline: NuGet.Client-Compliance
+      source: NuGet.Client-Official
+      trigger:
+        - dev
+
+jobs:
+  - job: Static Analysis
+    pool:
+      vmImage: windows-latest
+
+    steps:
+
+  - task: CredScan@2
+    inputs:
+      toolMajorVersion: "V2"
+
+  - task: PoliCheck@1
+    inputs:
+      inputType: "Basic"
+      targetType: "F"
+      targetArgument: "$(Build.SourcesDirectory)"
+      result: "PoliCheck.xml"
+
+  - task: SdtReport@1
+    displayName: "Generate Analysis Report"
+    inputs:
+      CredScan: true
+      PoliCheck: true
+      ToolLogsNotFoundAction: "Standard"
+
+  - task: TSAUpload@1
```

Comment:
Where are definition/documentation and list of supported tasks? TsaV2, CredScan, PoliCheck
Maybe I also need to same thing.

Reply:
Provided a link to the spec offline.